### PR TITLE
solution added

### PR DESCRIPTION
**Pull request by margarita-ignatieva**

### CODE REVIEW

File: src/main/java/com/cinema/shop/model/Ticket.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "tickets")
+public class Ticket {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToOne
```
Comment (Martynice) [inline src/main/java/com/cinema/shop/model/Ticket.java]: Many tickets can be connected to one session. Probably, it's better to use ManyToOne annotation

File: src/main/java/com/cinema/shop/service/impl/ShoppingCartServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    private ShoppingCartDao shoppingCartDao;
+    @Inject
+    private TicketDao ticketDao;
+
+    @Override
+    public void addSession(MovieSession movieSession, User user) {
+        Ticket ticket = new Ticket();
+        ticket.setMovieSession(movieSession);
+        ShoppingCart shoppingCart = shoppingCartDao.getByUser(user);
+        shoppingCart.getTickets().add(ticket);
```
Comment (Martynice) [inline src/main/java/com/cinema/shop/service/impl/ShoppingCartServiceImpl.java:27]: I'm not sure, but you add ticket, which has not been assosiated with DB yet. Probably, at first you should add it with ticketDao and then add to shopping cart. But I may be wrong

File: src/main/java/com/cinema/shop/model/ShoppingCart.java [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+import javax.persistence.MapsId;
+import javax.persistence.OneToMany;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "shoppingCarts")
+public class ShoppingCart {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
```
Comment (RomanDubovskyi) [inline src/main/java/com/cinema/shop/model/ShoppingCart.java]: is this annotation necessary when we have mapsId?

File: src/main/java/com/cinema/shop/model/Ticket.java (hunk 1 above)
Comment (RomanDubovskyi) [reply src/main/java/com/cinema/shop/model/Ticket.java]: > Many tickets can be connected to one session. Probably, it's better to use ManyToOne annotation

\+

File: src/main/java/com/cinema/shop/service/impl/ShoppingCartServiceImpl.java (hunk 2 above)
Comment (RomanDubovskyi) [reply src/main/java/com/cinema/shop/service/impl/ShoppingCartServiceImpl.java:27]: > I'm not sure, but you add ticket, which has not been assosiated with DB yet. Probably, at first you should add it with ticketDao and then add to shopping cart. But I may be wrong

\+

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**